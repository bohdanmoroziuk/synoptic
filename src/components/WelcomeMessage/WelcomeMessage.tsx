import { FunctionComponent } from 'react';
import { Text } from 'react-native';

const WelcomeMessage: FunctionComponent = () => (
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
);

export default WelcomeMessage;
