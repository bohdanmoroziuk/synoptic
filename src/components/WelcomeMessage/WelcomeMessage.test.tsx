import { render } from '@testing-library/react-native';

import WelcomeMessage from './WelcomeMessage';

describe('WelcomeMessage', () => {
  it('renders successfully', () => {
    const { toJSON } = render(<WelcomeMessage />);

    expect(toJSON()).toMatchSnapshot();
  });
});
