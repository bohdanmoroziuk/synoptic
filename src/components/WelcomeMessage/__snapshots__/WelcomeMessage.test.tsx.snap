// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeMessage renders successfully 1`] = `
<Text>
  Open up App.tsx to start working on your app!
</Text>
`;
